bash/.bash_greeter

```diff
@@ -1,4 +1,3 @@
-#!/bin/bash
```

Comment:
Why are all of these shebangs being removed?

Reply:
We have a new `pre-commit` hook that checks for files that are not executable but have shebangs.  These are such files.

Reply:
Thanks for the explanation. 👍